apps/web-tools/live-common-setup.ts

```diff
@@ -0,0 +1,61 @@
+import { setSupportedCurrencies } from "@ledgerhq/live-common/lib/currencies/index";
+import { setWalletAPIVersion } from "@ledgerhq/live-common/lib/wallet-api/version";
+import { WALLET_API_VERSION } from "@ledgerhq/live-common/lib/wallet-api/constants";
+
+setWalletAPIVersion(WALLET_API_VERSION);
+
+setSupportedCurrencies([
```

Comment:
A few chains may be missing here compared to LLD and LLM on develop

Reply:
( and i just took the same from common-tools )

Reply:
Which was probably outdated... it's on us so my bad, but this is an opportunity to update the list I think.
Another proof we'll want to unify this list of supported coins, coming soon with coin config :+1:

Reply:
Not blocking anyway, can be done later